packages/middleware-retry/src/retryMiddleware.ts

```diff
@@ -1,5 +1,5 @@
 import { HttpRequest, HttpResponse } from "@smithy/protocol-http";
-import { isServerError, isThrottlingError, isTransientError } from "@smithy/service-error-classification";
+import { isClockSkewError, isServerError, isThrottlingError, isTransientError } from "@smithy/service-error-classification";
```

Comment:
`isClockSkewError` is deprecated, and I think we should not use it because it will retry repeatedly for errors like `InvalidSignatureException`